### ISSUE TITLE
Parsing exception on invalid access_token

### ISSUE THREAD

**Issue by Zaszczyk** [thephpleague/oauth2-server#593]
I've configured ouath2 server with password grant and successfully generated a few access_tokens. Next I've implemented validating authenticated request (as it's described: http://oauth2.thephpleague.com/resource-server/securing-your-api/). It works fine for valid access_token. However when I change 1 char in token I got Runtime Exception:
-message: Error while decoding to JSON: unexpected character
-file: /vagrant/www/vendor/lcobucci/jwt/src/Parsing/Decoder.php
-line: 36

My request is:

```
GET /v1/feed/ HTTP/1.1
Host: api.xxx.dev
Authorization: Bearer xeyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImp0aSI6IjhlNDNlZDBjNjMzMDQxNjRmYzBmZTY0NWJkMDYyNTVmOWUzMzg4N2EyNWIzYjJlZDQ0OTBjM2U1NmY1MTMzMWIwYmRiODBhNWFhZTJmYmIxIn0.eyJhdWQiOiJmcSIsImp0aSI6IjhlNDNlZDBjNjMzMDQxNjRmYzBmZTY0NWJkMDYyNTVmOWUzMzg4N2EyNWIzYjJlZDQ0OTBjM2U1NmY1MTMzMWIwYmRiODBhNWFhZTJmYmIxIiwiaWF0IjoxNDY1NzYzODcwLCJuYmYiOjE0NjU3NjM4NzAsImV4cCI6MTc4MTI5NjY3MCwic3ViIjoiMSIsInNjb3BlcyI6WyJjb21wZXRpdGlvbiIsImNvbXBldGl0aW9uIl19.wBgj3dC0_mg_CoCattCLHwV-tTRdEIwtS9uqg6w84M22HCMNW_2dNCpq2H-XUsTseHLlHNlUw93Hcu9KFqIv0AxYx4GXf6q4iL8-ILd4f0sOfvxJeAbeoB87y4OI9GSrBlC9CuRQHbClQz5AFi_sdb1UyZJ9PwbEHucpbiFMMhw
Cache-Control: no-cache
Postman-Token: 1891a1ba-d533-80c6-c063-4e1756e72d28
```
I thought it should throw OAuthServerException, because access_token is fake, but the script break during parsing JSON.

**Comment by alexbilbie** [comment]
Thanks, will address in a patch later today

**Comment by alexbilbie** [comment]
This has been fixed by #605
